.circleci/config.yml

```diff
@@ -91,6 +104,7 @@ workflows:
       - unit-test-sdk:
           name: "Run Unit tests"
           xcode: "12.5.0"
+      - build-tests-for-simulator
```

Comment:
Why is this needed?

Reply:
Ah, I see it's building the test host tests. Can you clarify that in the name. maybe `build-test-host-tests` or something like that